specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/examples/StandardAssignments/GetStandardAssignment.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2024-08-01",
+    "resourceId": "providers/Microsoft.Management/managementGroups/contoso",
+    "standardAssignmentName": "1f3afdf9-d0c9-4c3d-847f-89da613e70a8"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Management/managementGroups/contoso/providers/Microsoft.Security/securityStandards/1f3afdf9-d0c9-4c3d-847f-89da613e70a8",
+        "name": "1f3afdf9-d0c9-4c3d-847f-89da613e70a8",
+        "type": "Microsoft.Security/standardAssignments",
+        "properties": {
+          "displayName": "Test exemption",
+          "description": "Exemption description",
+          "assignedStandard": {
+            "id": "/providers/Microsoft.Security/Standards/1f3afdf9-d0c9-4c3d-847f-89da613e70a8"
```

Comment:
the resourceType should be camelCased - also - should it be standards or securityStandards?